.github/workflows/coq-action.yml

```
@@ -8,10 +8,10 @@ on:
     branches:
       - '**'
   # to enable later (along with mathcomp/mathcomp-dev):
-  # schedule:
-  # # test master every day at 05:00 UTC
-  # # (mathcomp/mathcomp-dev is rebuilt every day at 04:00 Paris time)
-  # - cron: '0 5 * * *'
+  schedule:
+  # test master every day at 05:00 UTC
+  # (mathcomp/mathcomp-dev is rebuilt every day at 04:00 Paris time)
+  - cron: '0 5 * * *'
```

BTW, nitpick: I realize the indentation of two places of the `.yml` file is slightly buggy:

1. this `- cron` line should have indentation +2:

```suggestion
    - cron: '0 5 * * *'
```

2. and the end of the file as well:

```yml
    steps:
    - uses: actions/checkout@v2
    - uses: coq-community/docker-coq-action@v1
      with:
        opam_file: 'coq-mathcomp-odd-order.opam'
        custom_image: ${{ matrix.image }}
```

`=>`

```yml
    steps:
      - uses: actions/checkout@v2
      - uses: coq-community/docker-coq-action@v1
        with:
          opam_file: 'coq-mathcomp-odd-order.opam'
          custom_image: ${{ matrix.image }}
```

(I updated the docker-coq-action [README](https://github.com/coq-community/docker-coq-action#custom_image) accordingly)